loom/graphics/gfxQuadRenderer.cpp

```diff
@@ -106,29 +90,18 @@ void QuadRenderer::submit()
 
             if (!Graphics_IsGLStateValid(GFX_OPENGL_STATE_QUAD))
             {
-                // Select the shader.
-                if(sTinted)

```

Comment:
This was a pointless piece of code as both shaders were the same. Looks like there used to be a purpouse, now everything is tinted (with white, if nothing else).